templates/components/itilobject/timeline/form_task.html.twig

```diff
@@ -365,10 +365,17 @@
                         }
                      </script>
                      <div class="col-12">
-                        <button id="plan{{ rand }}" class="btn btn-outline-secondary d-block mb-2 text-truncate" onclick="showPlanUpdate{{ rand }}()" type="button">
+                        <button id="plan{{ rand }}" class="btn btn-outline-secondary text-truncate" onclick="showPlanUpdate{{ rand }}()" type="button">
                            <i class="fas fa-calendar"></i>
                            <span>{{ __('Plan this task') }}</span>
                         </button>
+                        {% if subitem.can(id, constant('UPDATE')) %}
```

Comment:
1. Button appears even if task is not planned. It should be visible only on already planned tasks.
2. When clicking on "Plan this task" button, the "unplan" button is still shown, maybe it should be hide.